restws.module

```diff
@@ -293,9 +383,9 @@ function restws_page_callback($resource, $page_callback = NULL) {
         // could poison the HTML page cache. A browser request to /node/1 could
         // suddenly return JSON if the cache was primed with this RESTWS
         // response.
-        if ($format && !isset($_COOKIE[session_name()]) && config_get('system.core', 'cache')) {
+        if ($format && !isset($_COOKIE[session_name()]) && variable_get('cache')) {
```

Comment:
I don't understand why you reverted this to variable_get. The D7 variable cache has been moved to config in `system_update_1003()`, and is accessible through the line you replaced:  `config_get('system.core', 'cache')`